tests/routes/get_offerers_test.py

```diff
@@ -281,6 +281,45 @@ def when_no_bank_information_for_offerer(self, app):
             assert response.json[0]['bic'] is None
             assert response.json[0]['iban'] is None
 
+        @clean_database
+        def test_returns_metadata(self, app):
+            # given
```

Comment:
```suggestion
            # Given
```
Idem pour les autres.